code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -702,7 +702,7 @@ var/list/datum/dna/hivemind_bank = list()
 //	spawn(10)	src.verbs += verb_path
 
 	src << SPAN_NOTICE("We stealthily sting [T].")
-	T << SPAN_WARNING("You feel a tiny prick.")
+//	T << SPAN_WARNING("You feel a tiny prick.")
```

Comment:
Please remove such things, if you don't need chat output than just remove it

Reply:
I'll keep that in mind in the future.